### PR TITLE
[core/ui] add seller address into orders component

### PR DESCRIPTION
**Pull request by vuvu2259**

### CODE REVIEW

File: core/ui/src/public/Orders/index.tsx [hunk 1]
```diff
@@ -34,10 +35,13 @@ export const Orders: React.FC<OrdersProps> = ({
   filters,
   style,
   defaultFilterName,
-  candyShop
+  candyShop,
+  sellerAddress
 }) => {
   const [sortedByOption, setSortedByOption] = useState(SORT_OPTIONS[0]);
   const [orders, setOrders] = useState<any[]>([]);
+  console.log(orders);
```
Comment (beepsaidblue) [inline core/ui/src/public/Orders/index.tsx:43]: could we remove

Comment (beepsaidblue) [inline core/ui/src/public/Orders/index.tsx:43]: could we remove

File: example/CandyShopContent.tsx [hunk 2]
```diff
@@ -55,12 +55,14 @@ export const CandyShopContent: React.FC<CandyShopContentProps> = ({ network }) =
         />
       </div>
 
+      <h1 style={{ textAlign: 'center', fontWeight: 'bold', marginBottom: 30 }}>Orders: 3fti...Cqoe</h1>
```
Comment (beepsaidblue) [inline example/CandyShopContent.tsx:58]: Get Orders By Address

Comment (beepsaidblue) [inline example/CandyShopContent.tsx:58]: Get Orders By Address

File: core/sdk/src/api/backend/OrderAPI.ts [hunk 3]
```diff
@@ -38,17 +39,20 @@ export async function fetchOrdersByStoreId(
...
+  if (identifiers && identifiers.length !== 0) {
     filterString = identifiers.reduce(
       (aggregated, identifier) =>
         aggregated +
         `&filterArr[]=${JSON.stringify({
           side: 1,
           status: 0,
-          identifier
+          identifier,
+          walletAddress: sellerAddress
```
Comment (beepsaidblue) [inline core/sdk/src/api/backend/OrderAPI.ts:50]: can we only pass walletAddress into the query if sellerAddress is specified

Comment (beepsaidblue) [inline core/sdk/src/api/backend/OrderAPI.ts:50]: can we only pass walletAddress into the query if sellerAddress is specified

File: core/sdk/src/api/backend/OrderAPI.ts [hunk 4]
```diff
@@ -38,17 +39,20 @@ export async function fetchOrdersByStoreId(
...
         })}`,
       ''
     );
+  } else {
+    filterString = `&filterArr[]=${JSON.stringify({ side: 1, status: 0, walletAddress: sellerAddress })}`;
```
Comment (beepsaidblue) [inline core/sdk/src/api/backend/OrderAPI.ts:55]: likewise here, only specify walletAddress if sellerAddress is specified

Comment (beepsaidblue) [inline core/sdk/src/api/backend/OrderAPI.ts:55]: likewise here, only specify walletAddress if sellerAddress is specified

File: core/ui/src/public/Orders/index.tsx (hunk 1 above)
Comment (vuvu2259) [reply core/ui/src/public/Orders/index.tsx:43]: Please help me remove it, i am on airplane

**Review by beepsaidblue — APPROVED** [review]

**Pull request merged**